### ISSUE TITLE
QuickUnlock window not Opening When restarting Keepass

### ISSUE THREAD

**Issue by Ronster99** [JanisEst/KeePassQuickUnlock#33]
About a year ago I tried to get this plugin to work but have never been successful. My plugin 2.4.0 is up to date. I have setup the QuickUnlock and added a three character password. I also setup Custom Entry in PartOf. When I save all and close then reopen the Keepass database I have never gotten the QuickUnLock window to pop up. I always have to enter my full database. Obviously I am doing something wrong but have tried installing this plugin about 6 or 7 times and have never been successful. Everything else on Keepass works great. Looking for some help.

**Comment by Ronster99** [comment]
For some reason I missed a very important point of how this plugin works. I had actually installed it correctly but for some reason I thought that once I installed it it would always ask for the short 3 or 4 character password even after closing Keepass or powering down my PC. WRONG! Yes I missed the part that says I must type-in the full password when first loading Keepass "THEN" if I click the "LOCK" (looks like a lock) in Keepass it minimizes Keepass, THEN when I click the minimized lock it asks me for my 3 character password. Simple. It works great now.
